lib/Github/Api/User.php

```diff
@@ -234,4 +234,23 @@ public function publicEvents($username)
     {
         return $this->get('/users/'.rawurlencode($username).'/events/public');
     }
+
+    /**
+     * List events performed by an authenticated user.
+     *
+     * @link https://docs.github.com/en/rest/reference/activity#list-events-for-the-authenticated-user
+     *
+     * @param string $username
+     * @param int    $page     the page number of the paginated result set
+     * @param int    $perPage  the number of results per page
+     *
+     * @return array
+     */
+    public function events($username, $page = 1, $perPage = 30)
```

Comment:
Please remove the `$page` and `$perPage` parameters, the [`ResultPager`](https://github.com/KnpLabs/php-github-api/blob/26a838ba251b9151c470a63b26a1ff45cebb5ba2/doc/result_pager.md) should be used to loop over all the results

Reply:
I just removed both parameters, thank you.